packages/fxa-auth-server/test/local/config/index.js

```diff
@@ -32,11 +32,10 @@ describe('Config', () => {
       assert.throws(() => {
         proxyquire(`${ROOT_DIR}/config`, {});
         // eslint-disable-next-line no-useless-escape
-      }, /Config \'[a-zA-Z.]+\' must be set in production/);
+      }, /Config \'[a-zA-Z._]+\' must be set in production/);
```

Comment:
`metrics.flow_id_key` is now the first thing on that list of required config values instead of `pushbox.key`